bench/Chainweb/Pact/Backend/Bench.hs

```diff
@@ -99,25 +97,23 @@ pactSqliteWithBench unsafe benchtorun =
 
 cpWithBench :: (CheckpointEnv -> C.Benchmark) -> C.Benchmark
 cpWithBench torun =
-    C.envWithCleanup setup teardown $ \ ~(NoopNFData (_,e,_)) ->
-                                        C.bgroup name (benches e)
+    C.envWithCleanup setup teardown $ \ ~(NoopNFData (_,e)) ->
+        C.bgroup name (benches e)
   where
     name = "batchedCheckpointer"
     cid = unsafeChainId 0
 
     initialBlockState = initBlockState $ genesisHeight v cid
 
     setup = do
-        (f, deleter) <- newTempFile
-        !sqliteEnv <- openSQLiteConnection f chainwebPragmas
+        let dbFile = "" {- temporary SQLite db -}
```

Comment:
This is the main change in this file: by passing "" as filename to `openSQLiteConnection`, the `deleter` isn't needed any more and can be removed from the resource environment.